src/components/Modals/ProjectCardModal/ProjectCardModal.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useState } from 'react';
+
+interface PropTypes {
+  open: boolean;
+  onClose: () => void;
+  onViewMoreProjects: () => void;
+}
+
+const ProjectCardModal: React.FC<PropTypes> = ({ open, onClose, onViewMoreProjects }) => {
+  const [answers, setAnswers] = useState<any[]>([
+    { id: 1, answerOne: "Sample answer one", answerTwo: "Sample answer two" }
+  ]);
+  const [answerOne, setAnswerOne] = useState<string>('');
+  const [answerTwo, setAnswerTwo] = useState<string>('');
+  const [isReviewMode, setIsReviewMode] = useState<boolean>(false);
+  const [submissionComplete, setSubmissionComplete] = useState<boolean>(false);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!answerOne || !answerTwo) {
+      alert("Please answer all the questions.");
+      return;
+    }
+    setIsReviewMode(true);
+  };
+
+  const handleFinalSubmit = () => {
+    const item = {
+      id: Math.floor(Math.random() * 1000),
+      answerOne,
+      answerTwo,
+    };
+    setAnswers((oldList) => {
+      const answers = [...oldList, item];
+      localStorage.setItem("answers", JSON.stringify(answers));
+      return answers;
+    });
+    setAnswerOne("");
+    setAnswerTwo("");
+    setIsReviewMode(false);
+    setSubmissionComplete(true);
+  };
+
+  useEffect(() => {
+    const itemsFromLocalStorage = localStorage.getItem("answers");
+    if (itemsFromLocalStorage) {
+      const parsedItemsFromLocalStorage = JSON.parse(itemsFromLocalStorage);
+      setAnswers(parsedItemsFromLocalStorage);
+    }
+  }, []);
+
+  return (
+    <div
+      className={`fixed inset-0 flex justify-center items-center transition-colors ${
+        open ? 'visible bg-black/20' : 'invisible'
+      }`}
+      onClick={onClose}
+    >
+      <div
+        className={`bg-white rounded-lg shadow p-6 transition-all max-w-md ${
+          open ? 'scale-100 opacity-100' : 'scale-110 opacity-0'
+        }`}
+        onClick={(e) => e.stopPropagation()}
+      >
+        <button
+          className="absolute top-2 right-2 py-1 px-2 border border-neutral-200 rounded-md text-gray-400 bg-white hover:text-grey-600"
+          onClick={onClose}
```

Comment:
I haven't loaded your app, does this also onClick also activate the onClose before. It's a hidden benign bug right now but you should have a stopPropafation on everything inside.

Or handle the outside modal click some other way like suggested before.

Reply:
I added e.stopPropagation() within the onClick handler to prevent the click event from propagating to the parent elements.